code-challenges.test.js

```diff
@@ -54,6 +102,50 @@ var vowelTester2 = "academy"
 var vowelTester3 = "challenges"
 // Expected output: 2
 
-
+// Writing the test starting with the describe() method.
+// This describe method lists the function name.
+describe("logFirstVowel", () => {
+    // Using the test() method to describe what the function will do.
+    test("takes in a string and logs the index of the first vowel", () => {
+        // Using 3 expect() methods for each of the variables that will invoke the logFirstVowel() function. Adding the .toEqual() matcher that checks the expected output.
+        expect(logFirstVowel(vowelTester1)).toEqual(1) 
+        expect(logFirstVowel(vowelTester2)).toEqual(0)
+        expect(logFirstVowel(vowelTester3)).toEqual(2)
+    })
+})
+// Ran 'yarn jest' and observed that the test correctly fails on its first iteration with the error 'logFirstVowel is not defined'
 
 // b) Create the function that makes the test pass.
+
+// Creating a function that takes a string as a parameter
+const logFirstVowel = (string) => {
+
+    // Trying a for loop
+    // If i is less than the length of the string, keep looping
+    for(i = 0; i < string.length; i++) {
+
+        // Using a regular expression to look for vowels
+        if(string.length === string.match(/[aeiou]/)) {
```

Comment:
will this conditional activate? What's an example of this conditional activating?